src/Components/Votes/Votes.jsx

```diff
@@ -1,5 +1,31 @@
 import React from "react";
+import { useState } from "react";
+import { updateVotesAdd, updateVotesDec } from "../../utils/API";
+import { Button } from "./Votes.styles";
 
-export default function Votes() {
-  return <div>Votes</div>;
-}
+const Votes = ({ votes, article_id }) => {
+  const [vote, setVote] = useState(0);
+
+  const voteUp = () => {
```

Comment:
This isn't DRY and should be refactored into one function that takes an arguments of the number to change votes by (eg 1 or -1)